packages/limit-service/lib/config.js

```diff
@@ -27,5 +27,7 @@ module.exports = {
             return result.count;
         }
     },
-    customThemes: {}
+    customThemes: {
+        allowlist: ['casper']
```

Comment:
As an artifact of the implementation I think it would be cool to create a way to quickly check and update this list (and link this this guide in here). Wdyt?